content/en/docs/advanced-tasks/adding-an-admission-webhook.md

```diff
@@ -70,3 +70,111 @@ Additionally, in an attempt to prevent operator from configuring the cluster int
 - Intercept requests that target all groups
 - Intercept requests that target the `operators.coreos.com` group
 - Intercept requests that target the `ValidatingWebhookConfigurations` or `MutatingWebhookConfigurations` resources
+
+### Conversion Webhook support
+
+OLM allows Operators to ship with [CRD Conversion](https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definition-versioning/#webhook-conversion) Webhooks when they only support the `AllNamespaces` installMode.
+Operator Authors may specify which of their owned CRDs rely on the Conversion Webhook by listing each of the CRD names in the `WebhookDefinitions.conversionCRDs` array within the operators `ClusterServiceVersion`.
```

Comment:
So effectively you're saying that the conversion webhook server must be served by a validating or mutating admission webhook server? Isn't that a little confusing, since a conversion webhook is not an admission webhook, and port/path can be different?

Reply:
Also, is there a preference for which type, validating or mutating, you should add `conversionCRDs` to if the same webhook server serves validating, mutating, and conversion webhooks? Can/should I put the list in both?

Reply:
@estroz it seems like we should add a new [type](https://github.com/operator-framework/api/blob/master/pkg/operators/v1alpha1/clusterserviceversion_types.go#L171) to the `webhookdefinition`.